src/data/programs/il_programs.ts

```diff
@@ -1,42 +1,106 @@
 export const IL_PROGRAMS = {
-  il_commonwealthEdison_applianceRebates: {
+  il_commonwealthEdison_comEdApplianceRebates: {
     name: {
       en: 'ComEd Appliance Rebates',
     },
     url: {
       en: 'https://www.comed.com/WaysToSave/ForYourHome/Pages/ApplianceRebates.aspx',
     },
   },
-  il_cornBeltEnergy_rebateProgram: {
+  il_commonwealthEdison_comEdSingleFamilyHomeEnergySavings: {
+    name: {
+      en: 'ComEd Single Family Home Energy Savings',
+    },
+    url: {
+      en: 'https://www.comed.com/ways-to-save/for-your-home/home-energy-savings/single-family',
+    },
+  },
+  il_commonwealthEdison_comEdElectricVehicleChargerAndInstallationRebate: {
+    name: {
+      en: 'ComEd Electric Vehicle Charger and Installation Rebate',
+    },
+    url: {
+      en: 'https://www.comed.com/about-us/clean-energy/electric-vehicle-charger-and-installation-rebate',
+    },
+  },
+  il_cornBeltEnergy_cornBeltEnergyRebateProgram: {
     name: {
       en: 'Corn Belt Energy Rebate Program',
     },
     url: {
       en: 'https://www.cornbeltenergy.com/programs-services/rebate-program',
     },
   },
-  il_stateOfIllinois_homeWeatherization: {
+  il_cornBeltEnergy_rebateProgram: {
     name: {
-      en: 'Illinois Home Weatherization Assistance Program',
+      en: 'Rebate Program',
+    },
+    url: {
+      en: 'https://www.cornbeltenergy.com/programs-services/rebate-program',
+    },
+  },
```

Comment:
Is this still used? It's weird to me that there's both this one and the one above

Reply:
Looks like there was one rogue row in the spreadsheet that need the program title updated - fixed ✅